src/CmsManager/CmsManagerInterface.php

```diff
@@ -106,12 +104,12 @@ public function setCurrentPage(PageInterface $page);
     /**
      * Returns the list of loaded block from the current http request.
      *
-     * @return BlockInterface[]
+     * @return PageBlockInterface[]
      */
     public function getBlocks();
 
     /**
-     * @param string $page
+     * @param int|string|null $page
```

Comment:
What is the `page` ? Do we mean pageId ?

Cause elsewhere it's typehinted `PageInterface $page`, so the name seems to be wrong here.

Reply:
But not a PageInterface, in this case it should be `pageId` instead of page, isn't it?

Reply:
So currently it's `pageIdOrPageUrl` ? That's why it's called `page` but it's really misleading with `PageInterface $page`.

Dunno if we could find a better name or if we should support PageInterface.

But it's not fully related to the PR so I won't block it

Reply:
I think there will be more PRs related to this, so we can change it later.